src/streamlink/plugins/onetv.py

```diff
@@ -72,6 +72,7 @@ def _get_streams(self):
         if self.is_live:
             self.logger.debug("Loading live stream for {0}...", self.channel)
 
+            self.session.http.headers = {"origin": "http://1tv.ru"}
```

Comment:
This will overwrite the existing headers, including any set on the command line. 
```python
# before
{'User-Agent': 'python-requests/2.22.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive'}
# after
{'origin': 'http://1tv.ru'}
```

You might need to change/remove the `User-Agent` header though as that might be affecting it too. 

```suggestion
            self.session.http.headers.update({"origin": "http://1tv.ru"})
```